dataverse-persistence/src/main/resources/db/migration/V23__addVocabularyValuesGroupingAndSortingByLocalizedStrings.sql

```diff
@@ -0,0 +1,196 @@
+ALTER TABLE controlledvocabularyvalue 
+ADD COLUMN displaygroup text;
+
+DELETE FROM controlledvocabularyvalue
+WHERE datasetfieldtype_id = 35;
+
+INSERT INTO controlledvocabularyvalue (id, displayorder, identifier, strvalue, datasetfieldtype_id, displaygroup) VALUES (55, 2, '', 'Abkhaz', 35, 'not_popular');
```

Comment:
`id` is generated, don't insert them explicitly. For `datasetfieldtype_id` get it dynamically.

Reply:
The id's are not generated - fixed values inserted in v2__bazeSchema.sql